src/legacy/core_plugins/kibana/public/management/sections/index_patterns/edit_index_pattern/edit_index_pattern.js

```diff
@@ -348,6 +348,7 @@ uiModules
 
         Promise.resolve($scope.indexPattern.destroy())
           .then(function() {
+            npStart.plugins.data.indexPatterns.clearCache();
             $location.url('/management/kibana/index_patterns');
```

Comment:
Wouldn't it be better if we clear the index pattern cache upon arrival to management?
Otherwise, if we get to that route from a different path, we'll have different behaviors.

Reply:
I'm just realizing we don't even need this clear cache to fix the bug because it already gets cleared when switching over to discover.

The index pattern listing in management is not affected by this because it's using the raw saved object client.

I will just revert this line.